client/executor_client.go

```diff
@@ -34,6 +34,8 @@ func (c *executorClient) Send(ctx context.Context, req *ExecutorRequest) (*Execu
 		resp.Resp, err = c.client.SubmitBatchTasks(ctx, req.SubmitBatchTasks())
 	case CmdCancelBatchTasks:
 		resp.Resp, err = c.client.CancelBatchTasks(ctx, req.CancelBatchTasks())
+	case CmdSuspendBatchTasks:
+		resp.Resp, err = c.client.SuspendBatchTasks(ctx, req.SuspendBatchTasks())
```

Comment:
What about using `pause` and `resume` task, which keeps the same style in DM and TiCDC.

Reply:
Good suggestion, I will change it later